armi/physics/fuelCycle/tests/test_fuelHandlers.py

```diff
@@ -661,41 +662,94 @@ def test_readMovesYaml(self):
         self.maxDiff = None
         expected = {
             1: [
-                AssemblyMove("LoadQueue", "009-045", [0.0, 12.0, 14.0, 15.0, 0.0], "outer fuel"),
+                AssemblyMove("LoadQueue", "009-045", [0.0, 0.12, 0.14, 0.15, 0.0], "outer fuel"),
                 AssemblyMove("009-045", "008-004"),
                 AssemblyMove("008-004", "007-001"),
                 AssemblyMove("007-001", "006-005"),
                 AssemblyMove("006-005", "SFP"),
                 AssemblyMove("009-045", "009-045", rotation=60.0),
-                AssemblyMove("LoadQueue", "010-046", [0.0, 12.0, 14.0, 15.0, 0.0], "outer fuel"),
+                AssemblyMove("LoadQueue", "010-046", [0.0, 0.12, 0.14, 0.15, 0.0], "outer fuel"),
                 AssemblyMove("010-046", "011-046"),
                 AssemblyMove("011-046", "012-046"),
                 AssemblyMove("012-046", "ExCore"),
             ],
             2: [
-                AssemblyMove("LoadQueue", "009-045", [0.0, 12.0, 14.0, 15.0, 0.0], "outer fuel"),
+                AssemblyMove("LoadQueue", "009-045", [0.0, 0.12, 0.14, 0.15, 0.0], "outer fuel"),
                 AssemblyMove("009-045", "008-004"),
                 AssemblyMove("008-004", "007-001"),
                 AssemblyMove("007-001", "006-005"),
                 AssemblyMove("006-005", "SFP"),
                 AssemblyMove("009-045", "009-045", rotation=60.0),
-                AssemblyMove("LoadQueue", "010-046", [0.0, 12.0, 14.0, 15.0, 0.0], "outer fuel"),
+                AssemblyMove("LoadQueue", "010-046", [0.0, 0.12, 0.14, 0.15, 0.0], "outer fuel"),
                 AssemblyMove("010-046", "011-046"),
                 AssemblyMove("011-046", "012-046"),
                 AssemblyMove("012-046", "ExCore"),
             ],
             3: [
-                AssemblyMove("LoadQueue", "009-045", [0.0, 12.0, 14.0, 15.0, 0.0], "outer fuel"),
+                AssemblyMove("LoadQueue", "009-045", [0.0, 0.12, 0.14, 0.15, 0.0], "outer fuel"),
                 AssemblyMove("009-045", "008-004"),
                 AssemblyMove("008-004", "007-001"),
                 AssemblyMove("007-001", "006-005"),
                 AssemblyMove("006-005", "SFP"),
                 AssemblyMove("009-045", "008-004"),
+                AssemblyMove("008-004", "009-045"),
                 AssemblyMove("007-001", "006-005"),
+                AssemblyMove("006-005", "007-001"),
             ],
         }
         self.assertEqual(moves, expected)
 
+    def test_performShuffleYamlIntegration(self):
+        fh = fuelHandlers.FuelHandler(self.o)
+        yaml_text = (
+            "sequence:\n"
+            "  1:\n"
+            "    - misloadSwap: [\"005-023\", \"006-029\"]\n"
+            "    - cascade: [\"igniter fuel\", \"009-045\", \"008-004\", \"007-001\", \"006-005\"]\n"
+            "      fuelEnrichment: [0, 0.12, 0.14, 0.15, 0]\n"
+            "    - extraRotations: {\"009-045\": 60}\n"
+        )
```

Comment:
This test is excellent. Thank you very much for adding it!

I cleaned up the readability by writing something like
```suggestion
        yaml_text = """
sequence:
  1:
    - misloadSwap: ["005-023", "006-029"]
    - cascade: ["igniter fuel", "009-045", "008-004", "007-001", "006-005"]
      fuelEnrichment: [0, 0.12, 0.14, 0.15, 0]
    - extraRotations: {"009-045": 60}
"""
```